docs/import/create_score_asset.js

```diff
@@ -1,13 +1,19 @@
-import {blockGroupTag, buildingCountTag, damageTag, geoidTag, incomeTag, snapPercentageTag, snapPopTag, sviTag, totalPopTag, tractTag} from '../property_names.js';
+import {blockGroupTag, damageTag, geoidTag, povertyHouseholdsTag, povertyPercentageTag, totalHouseholdsTag} from '../property_names.js';
 import {getBackupScoreAssetPath, getScoreAssetPath} from '../resources.js';
+
 import {computeAndSaveBounds} from './center.js';
 import {cdcGeoidKey, censusBlockGroupKey, censusGeoidKey, tigerGeoidKey} from './import_data_keys.js';
 
 export {createScoreAsset, setStatus};
-
 // For testing.
 export {backUpAssetAndStartTask};
 
+const TRACT_TAG = 'TRACT';
+const SVI_TAG = 'SVI';
+// Median household income in the past 12 months (in 2016 inflation-adjusted
+// dollars)
+const INCOME_TAG = 'MEDIAN INCOME';
```

Comment:
Is it always 2016 inflation-adjusted dollars or is it just the year you download from

Reply:
Just moved this from property_names.js. Removed.